web/src/app/admin/qualifications/qualifications.component.ts

```diff
@@ -1,14 +1,61 @@
-import {Component, OnInit} from '@angular/core';
+import { Component, OnInit} from '@angular/core';
+import { QualificationsServiceApp } from './shared/qualifications.serviceApp';
+import { QualificationViewModel } from '../../webapi/models/qualification-view-model';
 
 @Component({
-  selector: 'app-qualifications-list',
+  selector: 'qualifications',
   templateUrl: 'qualifications.component.html'
 })
 
 export class QualificationsComponent implements OnInit {
-  constructor() {
-  }
+ qualificationsModel: QualificationViewModel = {} as QualificationViewModel;
+  qualifications: QualificationViewModel[] = [] as QualificationViewModel[];
+  isCreateOrUpdate: boolean = true;
 
+  constructor(private QualificationsServiceApp: QualificationsServiceApp) {
```

Comment:
use camel casing for variable declarations everywhere

Reply:
done